source/manual/amazonopensearch.html.md

```diff
@@ -0,0 +1,64 @@
+---
+owner_slack: "#govuk-publishing-platform"
+title: Access Amazon OpenSearch Dashboard
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+---
+
+[AWS OpenSearch] is an open source, distributed search and analytics suite derived from Elasticsearch.
+
+## Prerequisites
+
+1. Access to a [GOV.UK EKS cluster] with admin role permissions has been configured and established.
+
+1. The [krelay] kubectl plugin has been installed.
+
+    ```sh
+    $ brew install knight42/tap/krelay
+    ```
+
+## Connect to the OpenSearch Dashboard
+
+1. List OpenSearch domain names:
+
+    ```sh
+    $ aws opensearch list-domain-names | jq -r '.DomainNames[]|.DomainName'
+    blue-elasticsearch6-domain
+    chat-engine
+    ```
+
+1. Get OpenSearch host name for `chat-engine`:
+
+    ```sh
+    $ aws opensearch describe-domain --domain-name chat-engine | jq -r '.DomainStatus.Endpoints.vpc'
+    vpc-chat-engine-m2k6ipw3klpaomdhrhhh6vo3de.eu-west-1.es.amazonaws.com
+    ```
+
+1. Forward the OpenSearch HTTPS port to your local machine:
+
+    ```sh
+    $ k relay host/vpc-chat-engine-m2k6ipw3klpaomdhrhhh6vo3de.eu-west-1.es.amazonaws.com 4443:443
+    ```
+
+1. Open <https://localhost:4443/_dashboards> in your browser. The TLS certificate will not match `localhost`, so navigate past the certificate warnings. In Chrome, you can set <chrome://flags/#allow-insecure-localhost> if you prefer.
+
+1. Log into the OpenSearch Dashboard web interface. The username and password for each environment can be found in [AWS Secrets Manager]. To find the `opensearch` secret name, use the command:
```

Comment:
As an aside, would we not need credentials if we connected with aws-sig-v4 (which is what AWS recommend: https://opensearch.org/docs/latest/clients/ruby/#connecting-to-amazon-opensearch-service)?

I'm thinking either we'd not need credentials and life could be easier - or it'd not work at all 😬. If it would work we might want to explore it to make it easier to see the dashboard.